plugins/TapTempo/TapTempoView.cpp

```diff
@@ -0,0 +1,147 @@
+/*
+ * TapTempoView.cpp - Plugin to count beats per minute
+ *
+ *
+ * Copyright (c) 2022 saker <sakertooth@gmail.com>
+ *
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+#include "TapTempoView.h"
+
+#include <QCheckBox>
+#include <QHBoxLayout>
+#include <QKeyEvent>
+#include <QLabel>
+#include <QPushButton>
+#include <QVBoxLayout>
+#include <QVariant>
+#include <QWidget>
+
+#include "Engine.h"
+#include "SamplePlayHandle.h"
+#include "Song.h"
+#include "TapTempo.h"
+
+namespace lmms::gui {
+TapTempoView::TapTempoView(ToolPlugin* plugin)
+	: ToolPluginView(plugin)
+	, m_plugin(dynamic_cast<TapTempo*>(plugin))
```

Comment:
```suggestion
TapTempoView::TapTempoView(TapTempo* plugin)
	: ToolPluginView(plugin)
	, m_plugin(plugin)
```
Should be able to do this - no more dynamic_cast